### DISCUSSION TITLE
Package not found

### DISCUSSION THREAD

**Discussion by Gluups** [community/community#88453] [New to GitHub]
Hello everybody,
I have a project of which the repository  has been cloned to Github.
Then I changed machine.
I try to clone the project from Github to the new machine.
Git tells me that it does not find the package ''System.ComponentModel.Annotations'.
Well, I found it. I do not know whether you will guess, but I found it in the Nuget packages.
How can I tell that to Git ?

I have the same problem for two more packages, I guess that probably the solution for the first package will help for the two other ones.

**Comment by OfficialMythicFrost** [comment]
@Gluups 
It seems like you need to restore the NuGet packages for your project. You can do this by running the following command in your project directory:

bash
dotnet restore

This command will download and install the necessary NuGet packages for your project, including 'System.ComponentModel.Annotations' and the other dependencies. After running this, try building your project again.

If you're not using .NET Core, you can use the following command instead:

bash
nuget restor

Make sure you have the necessary tools installed on your new machine, such as the .NET SDK or NuGet CLI, depending on your project type.
